ToDoListCoreDataTask/To-Do-List-App-Core-Data/Models/MigrationPolicy.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  MigrationPolicy.swift
```

Comment:
Whar is NoteMigrationPolicy, what was the use of this class?
If not used remove this file

Reply:
File removed.